components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/browser/BrowserModel.java

```diff
@@ -388,7 +388,9 @@ private void createGridImages()
      */
     private double getBarSizeInPx(double ratio)
     {
-        if(unitBarLength == null)
+        if (unitBarLength == null
+                || Double.isInfinite(getPixelsSizeX().getValue())
```

Comment:
I don't think a `BigResult` exception is thrown in that case, but a `NumberFormatException`, see https://www.openmicroscopy.org/qa2/qa/feedback/17673/ , that's why the check for finite.